templates/common/infra/terraform/core/gateway/apim-api/apim-api_variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "name" {
+  type        = string
+}
+
+variable "rg_name" {
+  description = "The name of the resource group to deploy resources into"
+  type        = string
+}
+
+variable "api_management_logger_id" {
+  description = "The name of the resource application insights"
+  type        = string
+}
+
+variable "web_front_end_url" {
+  description = "The url of the web"
+  type        = string
+}
+
+variable "apiName" {
+  description = "Resouce name to uniquely dentify this API within the API Management service instance"
+  type        = string
+}
+
+variable "apiDisplayName" {
+
+  description = "The Display Name of the API"
+  type        = string
+}
+
+variable "apiPath" {
+  description = "Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance. It is appended to the API endpoint base URL specified during the service instance creation to form a public URL for this API."
+  type        = string
+}
```

Comment:
The other variables use 'snake_case', but these use camelCase.  Can we be consistent and use snake for all?